src/utilities/index.ts

```diff
@@ -60,5 +60,34 @@ export const FormatBody = (message: string) => {
 }
 
 export function convertTimeStamp(timeStamp: string) {
-  return format(new Date(Number(timeStamp) * 1000), 'dd MMM yyyy | hh:mm a')
+  return format(new Date(Number(timeStamp) * 1000), "dd MMM yyyy | hh:mm a");
+}
+
+export async function httpRequest(url: RequestInfo, options?: RequestInit) {
```

Comment:
isn't `url` supposed to be `string`?? where is `RequestInfo`, 'RequestInit' even defined?? please define proper types and use them.